spec/fixtures/blank_repository2/test_plugins/report_filter.rb

```diff
@@ -0,0 +1,11 @@
+module Filter
+  module TestReportConfig
+    def self.filter_config(report_hash)
```

Comment:
We should change this to filter_report_hash for good measure.  That will avoid any potential collisions with modules that want to support both hooks (config and report hash)

Reply:
Thanks, updated accordingly.